.github/workflows/linters.yml

```diff
@@ -1,56 +1,70 @@
-# name: Linters
+name: Linters
 
-# on: pull_request
+on: pull_request
 
-# env:
-#   FORCE_COLOR: 1
+env:
+  FORCE_COLOR: 1
 
-# jobs:
-#   lighthouse:
-#     name: Lighthouse
-#     runs-on: ubuntu-22.04
-#     steps:
-#       - uses: actions/checkout@v3
-#       - uses: actions/setup-node@v3
-#         with:
-#           node-version: "18.x"
-#       - name: Setup Lighthouse
-#         run: npm install -g @lhci/cli@0.11.x
-#       - name: Lighthouse Report
-#         run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
-#   # webhint:
-#   #   name: Webhint
-#   #   runs-on: ubuntu-22.04
-#   #   steps:
-#   #     - uses: actions/checkout@v3
-#   #     - uses: actions/setup-node@v3
-#   #       with:
-#   #         node-version: "18.x"
-#   #     - name: Setup Webhint
-#   #       run: |
-#   #         npm install --save-dev hint@7.x
-#   #         [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.hintrc
-#   #     - name: Webhint Report
-#   #       run: npx hint .
```

Comment:
@cate339 Hello !

Your Eslinters are not running. Please fix the eslint before you can continue with merge. Try running 
`npm install --save-dev hint@7.x` then `npx hint . --fix`